### ISSUE TITLE
Accessibility 9.4.1.2 - disabled buttons not reachable

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#305]
# Bug

Die „deaktivierten“ Buttons wie z.B. der „Umfrage erstellen“-Button ist für den Screenreader nicht wahrnehmbar, da er nicht angesteuert und ausgegeben werden kann. („disabled“)

## SOLL-Zustand

Der Button sollte als schreibgeschützt programmatisch ausgezeichnet werden, z.B. durch Ergänzung eines aria-disabled=“true“. Evtl. muss das CSS auch dabei angepasst werden, um den „ausgegrauten“ Look zu erhalten. Mittels „aria-disabled=true“ kann der Button per Tastatur erreicht werden und der Screenreader gibt ihn aus mit z.B. „Umfrage erstellen Schalter nicht verfügbar“

Dies sollte für alle „deaktivierten“ Buttons in der Anwendung umgesetzt werden.

----------

# Bug

The “disabled” buttons, such as the “Create survey” button, are not visible to the screen reader as they cannot be activated and displayed. (“disabled”)

## Desired state

The button should be programmatically marked as read-only, e.g. by adding an aria-disabled=“true”. The CSS may also need to be adapted to maintain the “grayed out” look.
Using “aria-disabled=true”, the button can be accessed via the keyboard and the screen reader displays it with e.g. “Create survey button not available”.

This should be implemented for all “disabled” buttons in the application.
